tests/framework/installer/ceph_manifests.go

```diff
@@ -853,3 +911,258 @@ spec:
   displayName: ` + displayName + `
   store: ` + store
 }
+
+func (m *CephManifestsMaster) GetCSIRoles(namespace string) string {
+	return `
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-csi-rbd-attacher-sa
+  namespace: ` + namespace + `
+---
+kind: ClusterRole
```

Comment:
@rootfs to satisfy myself why we need clusterrole ? Cant we survive with `role` here?

Reply:
attacher needs to access cluster wide resources nodes and pv

Reply:
there should be two objects then. one for ClusterRole and one for Role. Its ok if a service account has multiple *Roles.

Reply:
yeah, having 2 roles attached to sa would be good choice in that case I believe